.github/workflows/ci.yaml

```diff
@@ -165,6 +168,9 @@ jobs:
         with:
           tool: cargo-nextest
       - uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: warm-windows
```

Comment:
I think we should avoid specying the shard key unless there are multiple jobs that use the same cache.

Reply:
Do we know if caches can be shared across builds on different architectures?

Reply:
I don't think it makes sense to share caches across architectures because the output is different for each architecture.